spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/actions/TestRemoveOrphanFilesAction.java

```diff
@@ -551,6 +554,48 @@ public void testManyLeafPartitions() throws InterruptedException {
     Assert.assertEquals("Rows must match", records, actualRecords);
   }
 
+  @Test
+  public void testHiddenPartitionPaths() throws InterruptedException {
+    Schema schema = new Schema(
+            optional(1, "c1", Types.IntegerType.get()),
+            optional(2, "_c2", Types.StringType.get()),
+            optional(3, "c3", Types.StringType.get())
```

Comment:
Nit: These are all over-indented. Code that is continued on the next line in the same expression should be. 4 spaces over.

There's an autoformatter that plugs into IntelliJ, and I believe eclipse and some other IDEs too:

Here's a link https://iceberg.apache.org/contribute/#setting-up-ide-and-code-style

But otherwise, these 3 lines should be 4 spaces over from the start of `Schema`. Like so
```java
    Schema schema = new Schema(
        optional(1, "c1", Types.IntegerType.get()),
        optional(2, "_c2", Types.StringType.get()),
        optional(3, "c3", Types.StringType.get()));
```

Reply:
There are a few other places that are 8 spaces over when they should be 4.